### ISSUE TITLE
GO component term for spore membrane

### ISSUE THREAD

**Issue by AndreaAuchincloss** [geneontology/go-ontology#20359]
Hello,
We (the Swiss-Prot section of UniProtKB) would like a new term for Spore membrane please. Right now it maps to cell membrane, which isn't correct. Spore walls have their own terms, the spore membrane should be the same.
So we would like new component term that is a child of GO:0005886, which could say:
The membrane surrounding the spore core (endospore core) that separates it from its external environment.

This would require removing the synonym "inner endospore membrane" from GO:0005886.

If you look at PMID 23202530 you can see that it is a separate membrane when mature
Thanks, Andrea

**Comment by pgaudet** [comment]
Doesn't the inner membrane correspond to the plasma membrane ? 
If not - do you want me to just remove the "inner endospore membrane" synonym, or should I move it to the new term ? 

Thanks, Pascale

**Comment by pgaudet** [comment]
Is this new term different from GO:0043593 endospore coat ? 
The layer in a bacterial endospore that lies under the exosporium, and is impermeable to many toxic molecules. The coat may also contain enzymes that are involved in endospore germination.
( this seems to be a type of cell wall - at least it's part:of the cell wall in GO).

**Comment by pgaudet** [comment]
Another question - should the new term be a child of 'endospore external encapsulating structure' ?

**Comment by AndreaAuchincloss** [comment]
Three answers to your 3 questions:
1) It does and doesn't correspond to the plasma membrane; it is the same space, but separate from it, and in SwissBioPics highlighting the spore membrane highlights the whole plasma membrane too in the diagrams, which is false, a protein in the spore membrane isn't necessarily in the plasma membrane.

2) Is this different from GO:0043593 endospore coat?  In the review I gave you it says;
Coat   A spore protective structure, which is made up of dozens of proteins. It is usually multilayered, consisting of inner and outer layers.
In Figure 1 the coat is shown as being the exterior-most layer, outside the membrane. 

3) Should it be a child of GO:0043591 endospore external encapsulating structure?
No, the definition of GO:0043591 talks about the inner membrane, and that is the component I am asking for

**Comment by pgaudet** [comment]
So, the name should be 'spore _inner_ membrane' ?

**Comment by pgaudet** [comment]
Sorry about all the questions !! 

The paper talks about 'inner forespore membrane’ - isn’t that clearer than just ‘spore’? I am concerned that spores are in fungi and dicty and bacteria, and just spore membrane may be misleading. 

Thanks, Pascale

**Comment by ValWood** [comment]
For history of why this term is not present
https://github.com/geneontology/go-ontology/issues/9807
specifically
https://github.com/geneontology/go-ontology/issues/9807#issuecomment-124900756


Does this work?

**Comment by AndreaAuchincloss** [comment]
No problem for the questions! That's your job. Spore inner membrane is also fine with me...
I read the argument against making a specific term, but to make a diagram via SwissBioPics, which I understand GO is hoping to use, mapping a protein that is specific to the spore membrane (but not found in the plasma membrane of a normal cell) is going to show up as both plasma membrane in the bacteria and in the spore. We are not planning on making the spore diagram separate from the bacteria diagram. Can you see my problem...

**Comment by pgaudet** [comment]
I'll make this 'only in bacteria', correct ?

**Comment by pgaudet** [comment]
+[Term]
+id: GO:0140549
+name: spore inner membrane
+namespace: cellular_component
+def: "The membrane surrounding the spore core (endospore core) that separates it from its external environment." [PMID:23202530]
+synonym: "spore membrane" BROAD []
+is_a: GO:0016020 ! membrane
+property_value: term_tracker_item https://github.com/geneontology/go-ontology/issues/20359 xsd:anyURI
+created_by: pg
+creation_date: 2020-11-12T10:00:15Z

**Comment by pgaudet** [comment]
It would be nice to describe better which terms are for bacteria, fungi, or others, as I suspect the spore structure is different ?

**Comment by AndreaAuchincloss** [comment]
Yes, let's make it bacteria-specific. I don't know enough about fungal spores (yet, I'll look into that ) to tell you, but I'll take a look and see if Marc/Ivo know more...
Thanks

**Comment by ValWood** [comment]
I think it will be confusing to have a spore membrane term only available to bacteria.

and I don't immediately see a difference to
 'prospore membrane' GO:0005628, it could be just semantics.
could you paste an image of the bacterial arrangement?

**Comment by pgaudet** [comment]
Fig 2 from PMID:23202530:

![image](https://user-images.githubusercontent.com/4782928/98942610-8eb56100-24ee-11eb-8c44-4ffb061bebab.png)

**Comment by ValWood** [comment]
OK that's tricky then. The bacterial spore is different.

I guess it's ok since fungi don't have an inner spore membrane.

**Comment by pgaudet** [comment]
Great. I'll add the taxon restriction.

**Comment by pgaudet** [comment]
@AndreaAuchincloss would it be OK to rename this term to endospore membrane ? Spore it too generic , see also #20502

**Comment by AndreaAuchincloss** [comment]
Seeing as you made the new term bacteria-specific I guess so...
